packages/grafana-ui/src/components/ColorPicker/ColorSwatch.tsx

```diff
@@ -14,12 +16,15 @@ export interface Props extends React.DOMAttributes<HTMLDivElement> {
   label?: string;
   variant?: ColorSwatchVariant;
   isSelected?: boolean;
+  ariaLabel?: string;
```

Comment:
Doesn't React.DOMAttributes<HTMLDivElement> include `aria-label`?

Reply:
It doesn't actually

Reply:
Maybe we could use `HTMLAttributes` instead?

Reply:
hmm...what might be the difference between the two? I think it feels much natural to write attributes in camelCase when working with react elements.

Reply:
I did a quick search in our solution for `DOMAttributes` compared to `HTMLAttributes`. It's 1 vs 44. I'm not saying that `HtmlAttributes` are more correct in this specific case but it seems like we used for the majority of the cases in our code base. 